src/shared/components/plots/BoxScatterPlot.tsx

```diff
@@ -546,8 +553,8 @@ export default class BoxScatterPlot<D extends IBaseBoxScatterPlotPoint> extends
         let paddingForLegend = 0;
 
         if (!this.props.horizontal) {
-            // more padding if vertical, because category labels extend to bottom
```

Comment:
this comment is still relevant, please leave it in in addition to your new comment